addons/im_livechat/static/src/core/public_web/thread_model_patch.js

```diff
@@ -75,6 +76,13 @@ patch(Thread.prototype, {
         return correspondent;
     },
 
+    _computeDisplayInSidebar() {
+        return (
+            this.discussAppCategory?.eq(this.store.discuss.livechatLookingForHelpCategory) ||
```

Comment:
Why check the category rather than just `livechat_status` directly?

Reply:
Category is created based on `has_access_livechat`. But I guess we can check the status, assuming that a live chat channel will only ever be known if we have access to the live chat (=category available).